Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSBeamCentreFinder.py

```diff
@@ -210,7 +210,7 @@ def PyExec(self):
             residueTB.append(self._calculate_residuals(sample_quartiles[MaskingQuadrant.Top],
                                                        sample_quartiles[MaskingQuadrant.Bottom]))
             if j == 0:
-                self.logger.notice("Itr {0}: ( {1}, {2} )  SX={3:.5g}  SY={4:.5g}".
+                self.logger.notice("Itr {0}: ( {1: .3f}, {2: .3f} )  SX={3:.5f}  SY={4:.5f}".
```

Comment:
Is the extra space in the centre values intended? This is the equivalent of prefixing the values with a space.

Reply:
I did not :man_facepalming: fixed now